### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by madpilot** [amberelectric/public-api#1] [Announcements]
Hello.

Welcome to the [Amber](https://amber.com.au) public API discussions forums. This repo doesn't have any code relating to our API - it will be used to discuss all things API related, post bug reports and show off your integrations!

If you have a question with the API, feel free to [start a new topic](https://github.com/amberelectric/public-api/discussions/new), and ask away.

If you find a bug, [create an issue](https://github.com/amberelectric/public-api/issues).

If you are looking for [documentation, it's here](https://app.amber.com.au/developers).

Be friendly, be kind. Let's go build things!

**Comment by AllTerrainAction** [comment]
Looks good certainly more straight forward than the past api, certainly recommend if anyone is hanging on to it they switch over.

Just simply works with the figures you need, data is straight forward. 9.5/10 for the dev team (0.5 because we need to see that it does what its supposed too!) hehe.

Many thanks for making this come out quicker than internally predicated.
